### ISSUE TITLE
v6.1.5 regression: callees include/exclude no longer match multi-line (chained) call expressions

### ISSUE THREAD

**Issue by esetnik** [edvardchen/eslint-plugin-i18next#162]
### Summary

v6.1.5 introduced a regression in `no-literal-string`: `callees.include` / `callees.exclude` patterns no longer match call expressions whose callee chain is formatted across multiple lines (the standard Prettier formatting for longer chains).

### Repro

Rule config:

```js
'i18next/no-literal-string': ['warn', { callees: { exclude: ['includes'] } }]
```

```js
// no warning in 6.1.4 or 6.1.5 — single line
msg.toLowerCase().includes('unmapped');

// no warning in 6.1.4, but WARNS in 6.1.5 — same code, multi-line chain
msg
  .toLowerCase()
  .includes('unmapped');
```

The only difference is formatting, so upgrading to 6.1.5 makes previously-clean code fail lint (breaking CI for anyone running with `--max-warnings 0`), and there is no code change that fixes it other than suppressing the rule — reformatting to one line is usually reverted by Prettier.

### Root cause

#159 (the fix for #138) changed how the dotted prefix is handled. Previously `generateFullMatchRegExp` produced:

```js
new RegExp(`(^|\\.)${source}$`)   // suffix match — newline-safe
```

Now `lib/rules/no-literal-string.js` pre-processes patterns via the new helper:

```js
function withDottedPrefix(patterns = []) {
  return patterns.map(item =>
    typeof item === 'string' ? `(?:.*\\.)?${item}` : item
  );
}
```

and `generateFullMatchRegExp` anchors with `^`. The resulting regex is `^(?:.*\.)?includes$`, matched against the callee's **source text** (`sourceCode.getText(callee)`). For a multi-line chain that text is:

```
msg
  .toLowerCase()
  .includes
```

Since `.` does not match newlines (the regex has no `s` flag), `.*` cannot span the chain, so the pattern never matches and the exclude is silently ignored.

The tests added in #159 only cover the new ellipsis case, so nothing caught this.

### Suggested fix

Make the prefix newline-tolerant in `withDottedPrefix`:

```js
`(?:[\\s\\S]*\\.)?${item}`
```

(or construct the regex with the `s` flag). Happy to open a PR with this plus a multi-line chained-callee test case if useful.

### Environment

- eslint-plugin-i18next: 6.1.5 (regression vs 6.1.4)
- Affects `callees` matching for `CallExpression`s; the same `withDottedPrefix` helper is also applied to `jsx-components` patterns, but component names can't contain newlines in practice, so `callees` is the user-visible breakage.
